Team9Project/src/com/indragie/cmput301as1/ClaimTabListener.java

```diff
@@ -0,0 +1,41 @@
+package com.indragie.cmput301as1;
+
+import android.app.ActionBar.Tab;
+import android.app.ActionBar.TabListener;
+import android.app.Activity;
+import android.app.Fragment;
+import android.app.FragmentTransaction;
+
+public class ClaimTabListener<T extends Fragment> implements TabListener {
+	private Fragment mFragment;
```

Comment:
We don't use the `m` prefix in the rest of the codebase so it should be consistent with the rest of the style. If the code style is like this because it was adapted from some other source, that should be cited.